packages/tradershub/src/screens/Address/Address.tsx

```diff
@@ -30,20 +31,28 @@ const Address = () => {
             },
             type: ACTION_TYPES.SET_ADDRESS,
         });
+        helpers.goToNextStep();
     };
 
     const initialValues = {
-        firstLineAddress: getSettings?.address_line_1 ?? state.firstLineAddress,
-        secondLineAddress: getSettings?.address_line_2 ?? state.secondLineAddress,
-        stateProvince: getSettings?.address_state ?? state.stateProvince,
-        townCity: getSettings?.address_city ?? state.townCity,
-        zipCode: getSettings?.address_postcode ?? state.zipCode,
+        firstLineAddress: getSettings?.address_line_1 ?? state.firstLineAddress ?? '',
```

Comment:
use ternary operator. nullish coalescing does not make sense here

Reply:
The first 2 is undefined at first, therefore using ?? is better i guess